s4e-backend/src/main/java/pl/cyfronet/s4e/controller/request/RegisterRequest.java

```diff
@@ -36,4 +36,6 @@
     @CountryCode
     @Schema(description = "User's ISO 3166 alpha-2 country code", example = "PL")
     private String country;
+
+    private String token;
```

Comment:
It's an optional part, which in other endpoints is passed as query param. I would put it in query param in this case too.